storage/core/src/common/internal/Helpers.ts

```diff
@@ -15,6 +15,17 @@ import {
   UrlTransferInput,
 } from "../Interfaces";
 
+export const defaultExpiresInSeconds = 60 * 60;
+export type ExpiryOptions =
```

Comment:
This should be exported from the package because it is a part of the public interface. This could go into storage\core\src\server\Interfaces.ts file

Reply:
done (both comments)